dashboard/app/views/teacher_dashboard/show.html.haml

```diff
@@ -2,7 +2,7 @@
   teacher_dashboard_data = {}
   teacher_dashboard_data[:studioUrlPrefix] = CDO.studio_url('', CDO.default_scheme)
   teacher_dashboard_data[:pegasusUrlPrefix] = CDO.code_org_url('', CDO.default_scheme)
-  teacher_dashboard_data[:visibleSections] = @visible_sections
+  teacher_dashboard_data[:sections] = @sections
```

Comment:
this is likely to cause some errors during and for several hours after the deploy, when the server updates before the JS running in the user's browser does. The best practice would be to also keep setting `teacher_dashboard_data[:visibleSections]` in this PR, wait 1-2 days, and then remove it once users JS has had a chance to update.

This might be a bit over-the-top since the user can just refresh the page to solve the problem... but we don't have very good JS error monitoring right now so it seems worth taking the extra step.

Reply:
not over-the-top! i'll leave a reminder to remove @visible_sections in a few days so i don't forget 😄